src/components/Picture.astro

```diff
@@ -0,0 +1,30 @@
+---
+type Props = {
+  className?: string
+  src: string
+  alt: string
```

Comment:
let's add another prop for the loading attr for the images, I'm thinking that we may need some images to apply lazy load. See: https://developer.mozilla.org/en-US/docs/Web/API/HTMLImageElement/loading